cli/cmd/lql.go

```diff
@@ -50,47 +50,72 @@ var (
 
 	// queryCmd represents the lql parent command
 	queryCmd = &cobra.Command{
-		Aliases: []string{"lql"},
 		Use:     "query",
+		Aliases: []string{"lql", "queries"},
 		Short:   "run and manage queries",
-		Long: `Run and manage queries.
+		Long: `Run and manage Lacework Query Language (LQL) queries.
 
-NOTE: This feature is not yet available!`,
+To provide customizable specification of datasets, Lacework provides the Lacework
+Query Language (LQL). LQL is a human-readable text syntax for specifying selection,
+filtering, and manipulation of data.
+
+Currently, Lacework has introduced LQL for configuration of AWS CloudTrail policies
+and queries. This means you can use LQL to customize AWS CloudTrail policies only.
+For all other policies, use the previous existing methods.
+
+Lacework ships a set of default LQL queries that are available in your account.
+
+For more information about LQL, visit:
+
+   https://support.lacework.com/hc/en-us/articles/4402301824403-LQL-Overview
+
+To view all LQL queries in your Lacework account.
+
+    lacework lql ls
+
+To show a query.
+
+    lacework query show <query_id>
+
+To execute a query.
+
+    lacework query run <query_id>
+
+** NOTE: LQL syntax may change. **
+`,
 	}
 
 	// queryRunCmd represents the lql run command
 	queryRunCmd = &cobra.Command{
 		Aliases: []string{"execute"},
 		Use:     "run [query_id]",
 		Short:   "run a query",
-		Long: `Run a query.
-
-Run a query via editor:
+		Long: `Run an LQL query via editor:
 
-	$ lacework query run --range today
+    lacework query run --range today
 
 Run a query via ID (uses active profile):
 
-	$ lacework query run MyQuery --range
+    lacework query run MyQuery --range
```

Comment:
We should either omit `--range` here `lacework query run MyQuery` or specify a valid range `lacework query run MyQuery --range today`.  Since we're already using `--range today` above, perhaps we should illustrate the use of `--start` and `--end`.  For instance: `lacework query run MyQuery --start "-1w@w" --end "@w"` ?

Reply:
I can change it 👍🏽